README.md

```diff
@@ -2,3 +2,8 @@
 
 [![Join the chat at https://gitter.im/stxnext-csr/volontulo](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/stxnext-csr/volontulo?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 Portal łączący wolontariuszy, organizacje i instytucje.
+Web portal for collaboration of community volunteers with organizations and institutions
+
+## Instalation
+
+TODO
```

Comment:
There is not so much additional value in this TODO. Have You already installed app?